build_runner/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 0.8.2
 
+- Allow passing multiple `--output` options. Each option will be split on
+  `:`. The first value will be the output directory, the second value will
+  be the root input directory. If no delimeter is provided, all resources
+  will be merged.
```

Comment:
Not really merged right? They are copied under their original top level directory names.